SeaPublicWebsite/Helpers/UserFlow/PageLinker.cs

```diff
@@ -0,0 +1,797 @@
+﻿using System;
```

Comment:
Maybe move this file into \Services and call it `WorkflowService`
A `Helpers` folder is sometimes necessary but should be kept as small as possible, otherwise it just becomes a big dumping ground for loads of stuff